R/robust_estimation.R

```diff
@@ -40,7 +40,9 @@
 #'     vcov_type = "CR2",
 #'     vcov_args = list(cluster = iris$cluster)
 #'   )
-#' }}
+#' }
+#' }
+#'
```

Comment:
I don't like this styling rule, as it adds unnecessary empty lines to the examples in help-pages.